docs/concepts/event-driven-architecture.md

```diff
@@ -0,0 +1,82 @@
+---
+id: event-driven-architecture
+title: Event-Driven Architecture with Amplication
+sidebar_label: Event-Driven Architecture
+slug: /concepts/event-driven-architecture
+---
+
+# Event-Driven Architecture with Amplication
+
+Event-driven architecture is a design pattern that enables loose coupling between microservices. In an event-driven system, services communicate with each other by producing and consuming _events_. This allows for real-time processing and reaction to events.
+
+Amplication provides built-in support for event-driven architectures through the use of _message brokers_ and _topics_. By leveraging these features, you can easily build scalable and responsive microservices.
+
+:::tip
+Amplication's [Break The Monolith](/break-the-monolith) feature lets you easily update monolithic architectures into a modern, scalable, and maintainable microservice architecture.
+:::
+
+## Key Concepts
+
+Before getting started, let's define some important concepts and terms related to event-driven architecture:
+
+- **Microservice**: A microservice is a small, independent service that focuses on a single business capability. Microservices are designed to be loosely coupled, allowing them to be developed, deployed, and scaled independently. In an event-driven architecture, microservices communicate with each other through events and message brokers.
+
+- **Message Broker**: A message broker is a software component that facilitates communication between services in an event-driven architecture. It receives messages from producers, stores them, and routes them to the appropriate consumers based on predefined rules or subscriptions.
+
+- **Topic**: A topic is a named stream or category of events. Producers publish messages to topics, and consumers subscribe to topics to receive messages. Topics provide a logical grouping and enable multiple consumers to independently process events from the same topic.
+
+- **Producer**: A producer is a service or component that generates and publishes events to one or more topics. Producers are responsible for creating messages and sending them to the message broker.
+
+- **Consumer**: A consumer is a service or component that subscribes to one or more topics and processes the received events. Consumers read messages from the subscribed topics and perform actions based on the content of the messages.
+
+## Enabling Event-Driven Microservices
+
+To implement event-driven architecture in your Amplication project, you'll need to:
+
+1. Create a message broker
+2. Define topics
+3. Connect services to the message broker
```

Comment:
@dericksozo - Connect services to the message broker and the relevant topic/s (?)